api/PerformanceResourceTiming.json

```
@@ -1004,9 +1012,21 @@
             "web-features:resource-timing"
           ],
           "support": {
-            "chrome": {
-              "version_added": "43"
-            },
+            "chrome": [
+              {
+                "version_added": "133"
+              },
+              {
+                "version_added": "115",
+                "version_removed": "133",
+                "partial_implementation": true,
+                "notes": "`responseStart` used the final response time even if an interim response was available ([chromestatus](https://chromestatus.com/feature/5158830722514944))."
```

Let's make it more concise (we don't usually link to Chrome Platform Status in notes):

```suggestion
                "notes": "Returns the time of the final response, even if an interim response was available."
```